src/main/java/thePackmaster/cards/clawpack/TwinClaw.java

```diff
@@ -0,0 +1,46 @@
+package thePackmaster.cards.clawpack;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainEnergyAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.combat.ClawEffect;
+import thePackmaster.actions.metapack.WaitMoreAction;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.CLAW;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class TwinClaw extends AbstractClawCard {
+    public final static String ID = makeID("TwinClaw");
+
+    public TwinClaw() {
+        super(ID, 1, CardType.ATTACK, CardRarity.COMMON, CardTarget.ENEMY);
+        baseDamage = 3;
+        baseMagicNumber = magicNumber = 2;
+        tags.add(CLAW);
+
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        if (m != null) {
+            this.addToBot(new VFXAction(new ClawEffect(m.hb.cX, m.hb.cY, Color.FIREBRICK, Color.WHITE), 0.1F));
+            addToBot(new WaitMoreAction(0.2F));
+            this.addToBot(new VFXAction(new ClawEffect(m.hb.cX, m.hb.cY, Color.RED, Color.WHITE), 0.1F));
+        }
+        this.addToBot(new DamageAction(m, new DamageInfo(p, this.damage, DamageInfo.DamageType.NORMAL), AbstractGameAction.AttackEffect.NONE));
+        this.addToBot(new DamageAction(m, new DamageInfo(p, this.damage, DamageInfo.DamageType.NORMAL), AbstractGameAction.AttackEffect.NONE));
+
+        ClawUp(magicNumber);
+    }
+
+    public void upp() {
+        upgradeMagicNumber(2);
```

Comment:
Is upgrading magic number instead of damage for Twin Claw intentional? I ask because I was expecting this to get +2 damage on upgrade like Twin Strike does, instead of increasing claw scaling.

Totally fine if that's how you want the card to work, just commenting in case this was an oversight.

Reply:
Yeah this was intended, as the idea being that by default, if it was truly "two claws" it should be +4 sharp but that'd be too strong of a card by base.  So when upgraded its actually just two claws.